porch/engine/pkg/engine/kuberuntime.go

```diff
@@ -0,0 +1,190 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package engine
```

Comment:
I'd prefer to move this logic out of the engine. We could move it into:
* apiserver module (apiserver is aware that it is running in k8s)
* func gRPC server (my preference)
* or into a separate, dedicated, gRPC server (would work for me too)

Reply:
Moved it into func gRPC server